public/docs/_examples/attribute-directives/dart/lib/app_component.html

```diff
@@ -0,0 +1,12 @@
+<!-- #docregion -->
+<h1>My First Attribute Directive</h1>
+<h4>Pick a highlight color</h4>
+<div>
+    <input type="radio" name="colors" (click)="color='lightgreen'">Green
+    <input type="radio" name="colors" (click)="color='yellow'">Yellow
+    <input type="radio" name="colors" (click)="color='cyan'">Cyan
+</div>
+<p><span [my-highlight]="color">Highlight me!</span></p>
+<p><span [my-highlight]="color" [default-color]="'violet'">
```

Comment:
Yes it does. Anything between the double quotes is evaluated by the angular engine. If it's not in single quotes then it would look for a String object name violet